paddle/fluid/framework/ir/CMakeLists.txt

```diff
@@ -92,6 +92,14 @@ if(WITH_MKLDNN)
     pass_library(cpu_quantize_squash_pass inference mkldnn)
 endif()
 
+if(WITH_NGRAPH)
+    cc_library(ngraph_subgraph_pass SRCS ngraph_subgraph_pass.cc DEPS ngraph_bridge
+      analysis_helper subgraph_detector graph_pattern_detector pass fuse_pass_base ${op_library_DEPS})
+    set(pass_file ${PADDLE_BINARY_DIR}/paddle/fluid/inference/api/paddle_inference_pass.h)
+    file(APPEND ${pass_file} "USE_PASS(ngraph_subgraph_pass);\n")
+    set(INFER_IR_PASSES ${INFER_IR_PASSES} ngraph_subgraph_pass CACHE INTERNAL "")
```

Comment:
Why not use 
```
pass_library(ngraph_subgraph_pass inference ngraph)
```

Reply:
Yes, your suggestion seems to be fine, Can I take this suggestion into consideration, and do it in the next pull-request?

Reply:
it seems it won't work as some dependency will be missing

Reply:
> Can I take this suggestion into consideration, and do it in the next pull-request?

Yes, please do it in the next PR.